content/rc/cloud-accounts/iam-resources/terraform.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: Create IAM resources using Terraform
+linkTitle: Terraform
+Weight: $weight
+categories: ["RC"]
+alwaysopen: false
+aliases: /rc/how-to/creating-iam-resources/terraform/
+         /rc/how-to/creating-iam-resources/terraform.md
+         /rc/cloud-accounts/iam-resources/terraform/
+         /rc/cloud-accounts/iam-resources/terraform.md
+
+---
+You can use [HashiCorp Terraform](https://www.terraform.io/intro/index.html) to create identity and access (IAM) resources to support AWS cloud account access to Redis Enterprise Cloud subscriptions.
```

Comment:
Should this be changed from "create identity and access (IAM) resources" to "create identity and access management (IAM) resources"?

Reply:
Fixed.